apps/frontend/src/app/poll/ical/ical.component.html

```diff
@@ -0,0 +1,74 @@
+<ngbx-modal #modal [back]="['..']">
+  <ng-container modal-title>
+    iCal Export
+  </ng-container>
+  <ng-container modal-body>
+    <h5>
+      Options
+    </h5>
+    <ngbx-validator-form [model]="config"></ngbx-validator-form>
+    <h5>
+      Preview
+    </h5>
+    <div class="card">
+      <div class="card-body">
+        <table>
+          <tr>
+            <th>
+              <i class="bi-square-fill text-primary-emphasis"></i>
+            </th>
+            <td>
+              <h5 class="card-title">
+                {{ config.customTitle ?? poll?.title }}:
```

Comment:
```suggestion
                {{ config.customTitle ?? poll?.title }}
```
If I type a title and remove it afterwards no name for the customTitle is given.